### ISSUE TITLE
Suggestions: small cleanup

### ISSUE THREAD

**Issue by Vadiml1024** [LibreSolar/bms-firmware#83]
https://github.com/LibreSolar/bms-firmware/blob/8f6a9492b2740a1759a6a7b65452ecda5bd482f1/drivers/bms_ic/bq769x2/bq769x2.c#L468

This is probably not needed

**Comment by martinjaeger** [comment]
True. It's a leftover from the same refactoring as the other bug you found. We used to have a cached (actual) config and the desired config provided as a parameter. Will provide a fix soon.
